Dokumentationen/test/test_cases.adoc

```diff
@@ -378,12 +378,16 @@ a| *
 |===
 
 |*Beschreibung* |
+Die Werte der Voreinstellungen (Studiengang, Uhrzeit, Datum) und die Auswahl der Details von Ausgaben (Dozentenwoche: Dozent, Datum; Statistik: KW, Semester) und alle Filtermöglichkeiten werden aus einem festgelegten Wertebereich gewählt. Realisiert wird dies durch Dropdown-Menüs und einen Datepicker. Es wird erwartet, dass die Voreinstellungen nicht durch Freitextfelder erfasst werden können. (manueller Test)
```

Comment:
Wollen wir das "(manueller Test)" immer mit in die Beschreibung aufnehmen? Ich hatte das bisher immer nur im Titel stehen.

Reply:
Eins von beiden sollten wir auf jeden Fall machen! Von mir aus können wir es auch mit in den Titel schreiben, aber ich dachte mir, dass wir dann bei den nicht-manuellen Tests auch eine Art des Testes angeben müssen, worauf wir ja aber einfach verzichten kann.